code/src/java/pcgen/persistence/SourceFileLoader.java

```diff
@@ -1112,7 +1114,7 @@ private void logError(String message, Throwable e)
 	 * @param sources
 	 *            The list of potential sources to be added.
 	 */
-	private void addQualifiedSources(Campaign c, ListKey<CampaignSourceEntry> lk)
+	private void addQualifiedSources(CDOMObject c, ListKey<CampaignSourceEntry> lk)
```

Comment:
This is *designed for* Campaigns, not CDOMObjects in general.  So this is actually now communicating incorrectly to a user of the method.  It is communicating false information by being made more generic.